arch/arm/acle_intrins.h

```diff
@@ -0,0 +1,27 @@
+#ifndef ARM_ACLE_INTRINS_H
```

Comment:
We should consider moving parts of `fallback_builtins.h` to `arch/x86_intrins.h` at some point.